sycl/doc/GetStartedGuide.md

```diff
@@ -48,6 +48,9 @@ and a wide range of compute accelerators such as GPU and FPGA.
   * libhwloc-dev or hwloc-devel package on linux
 * C++ compiler
   * See LLVM's [host compiler toolchain requirements](https://github.com/intel/llvm/blob/sycl/llvm/docs/GettingStarted.rst#host-c-toolchain-both-compiler-and-standard-library)
+* `zstd` version 1.5.6 or later (Optional)
+   * For Ubuntu, you can obtain it by `apt install libzstd-dev`
```

Comment:
Aah, you're right. Now that I look at it, even zstd 1.4.8 (supported by Ubuntu 22.04) will work as the zstd APIs used in LLVM and SYCL RT haven't changed since v1.4.8. In intel/llvm CI, we use v1.4.8 on Ubuntu 22.04 docker images and it seems to fine there.